### ISSUE TITLE
[Usability] A11y_Manage.IIS.Net_Webserver Ip Restriction_Screen Reader: After checking  the checkbox some (....) symbol is coming,So NVDA should announce the progress status for this	

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#193]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Chrome (68.0.3440.106)
Screen Reader

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Switch on the NVDA
4.Tab till Microsoft IIS on top navigation bar and press enter
5.Tab till Web server and press enter
6.Navigate up to IP Restriction and press enter
7.Tab till check box and check it
8.check after checking the check box Nvda is reading the progress status or not

Actual Result:
After checking  the checkbox some (....) symbol is coming, So NVDA should announce the progress status for this	

Expected Result:
Screen Reader should announce the progress status for this

User Impact:
Screen Reader users will not understand what is going on background 

Reference video:
[Bug_193_NVDA is not reading the progress status.pptx](https://github.com/Microsoft/IIS.WebManager/files/2335767/Bug_193_NVDA.is.not.reading.the.progress.status.pptx)

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yUsable; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #C+E-H; #.NETWebsites; #Chrome; #NVDA
